kafka_utils/kafka_check/commands/command.py

```diff
@@ -58,7 +59,7 @@ def run(self, cluster_config: ClusterConfig, args: argparse.Namespace) -> tuple[
             if args.controller_only and not is_controller(self.zk, args.broker_id):
                 terminate(
                     status_code.OK,
-                    cast(dict[str, Any], prepare_terminate_message(
+                    cast(Dict[str, Any], prepare_terminate_message(
```

Comment:
What's the reason for switching to `typing.Dict` here?

Reply:
So `mypy` understands this... but the actual code in python 3.7 doesn't. I ran into an error in the acceptance tests on a cast I added in this branch. And then I realized that I probably broke this code by using this before. So I went back and fixed all `cast` usages.